source/comtypesMonkeyPatches.py

```diff
@@ -3,6 +3,47 @@
 #This file is covered by the GNU General Public License.
 #See the file COPYING for more details.
 
+import ctypes
+import _ctypes
+
+# A version of ctypes.WINFUNCTYPE 
+# that produces a WinFunctionType class whos instance will convert COMError into a CallCancelled exception when called as a function.

```

Comment:
I'm no native English speaker, but I'm pretty sure whos should be whose.

Reply:
I have confirmed we still need all fixes for comtypes, except for the one to support VT_BYREF. However, there code is slightly different to hours, so I am reluctant to remove ours, especially in this pr, without first finding a good example to test with.
From memory the only thing we ever called that took values byref was MS Word's Window.getPoint.